src/Core/TempFolder.php

```diff
@@ -18,9 +18,15 @@ class TempFolder
     public static function getTempFolder($base)
     {
         $parent = static::getTempParentFolder($base);
+        $user = static::getTempFolderUsername();
+
+        // Append php version to username folder
+        if (Environment::getEnv('TEMP_FOLDER_VERSION')) {
```

Comment:
```suggestion
        if (Environment::getEnv('SS_TEMP_FOLDER_INCLUDES_PHP_VERSION')) {
```
Just to make it clearer what this controls and add the standard `SS_` prefix.

Should maybe even be `SS_PROJECT_TEMP_FOLDER_INCLUDES_PHP_VERSION` since it doesn't affect the temp folder if it's in tmp/... but that's pretty long so probably we can let documentation cover that detail.